app/lib/screens/faq/Item.dart

```diff
@@ -0,0 +1,30 @@
+// ignore_for_file: prefer_const_constructors
```

Comment:
I prefer to call them "Details" as "ListItem" and "Item" are not easy to distinguish. Let's please refer to the single object Screens as "XYZDetailsScreen". Please rename the file and the component accordingly.